packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -175,7 +175,7 @@ const AuthDefault = ({
             text={{
               body: shouldLogin
                 ? 'Not a member yet?'
-                : 'Already a daily.dev member?',
+                : 'Already using daily.dev?',
               button: shouldLogin ? 'Sign up' : 'Log in',
```

Comment:
Should we say free forever for sign up here as well?

Reply:
No it's only where it also says it's free